src/eXpansion/Bundle/LocalRecords/Model/Base/Record.php

```diff
@@ -2,26 +2,25 @@
 
 namespace eXpansion\Bundle\LocalRecords\Model\Base;
 
-use \DateTime;
```

Comment:
You should never touch this files, theye are autgenerated. It puts to confusion.